rca/project_styleguide/templates/patterns/molecules/accordion/accordion.html

```diff
@@ -40,7 +40,7 @@ <h5 class="accordion__heading heading heading--five">{{ accordion.heading }}</h5
                     {{ accordion.introduction }}
                 </div>
             {% endif %}
-            {{ accordion.body|richtext }}
+            {{ accordion.body|richtext|linebreaks }}
```

Comment:
Is `linebreaks` needed for existing data...?

Reply:
Linebreaks is actually added to handle these new fields, as they aren't rich text. But the accordion is used allover the place with rich text fields.